compression/zinfo.h

```diff
@@ -76,74 +76,52 @@ typedef int64_t offset_t;
 #define BLOB_HEADER_SIZE (4 + 8)
 
 
-enum 
-{
+enum {
```

Comment:
Another change I feel necessary but didn't make is changing `zinfo.c|h` to `gzip_zinfo.c|h`. They're only meant to work with gzip right?

Reply:
fixed